.github/workflows/monitor-stable-releases.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: Monitor-stable-releases
+
+#on:
+#  release:
+#    types: [released]
+
+
+concurrency: # This keeps multiple instances of the job from running concurrently for the same ref and event type.
+  group: release-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  update-stable-agents-metadata:
+    name: update-stable-agents-metadata
+    if: ${{ github.ref == 'refs/heads/master' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+      - name: Update latest packaged version
+        id: create-latest-versions-metadata
+        run: |
+          sh .github/scripts/update-latests-per-major-version.sh $(cat packaging/version)
+      - name: Commit Changes
+        id: commit
+        if: steps.create-latest-versions-metadata.outputs.run == 'true'
+        env:
+          GITHUB_TOKEN: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+        run: |
+          git config user.name "netdatabot"
+          git config user.email "bot@netdata.cloud"
+          git add packaging/agent-metadata/stable
+          git commit -m "[ci skip] Update the latest major version published"
```

Comment:
This doesn’t actually currently result in any actual changes on the master branch, because stuff never gets pushed.

Reply:
thanks forgot it (also comment above)

Reply:
address in https://github.com/netdata/netdata/pull/16366/commits/472a496c6dbeab6c2350df945e67afbb09a4cf6f